pkg/provider/kubernetes/crd/traefikio/v1alpha1/ingressroute.go

```diff
@@ -126,6 +126,11 @@ type LoadBalancerSpec struct {
 	// The Kubernetes Service itself does load-balance to the pods.
 	// By default, NativeLB is false.
 	NativeLB bool `json:"nativeLB,omitempty"`
+	// NodePortLB controls, when creating the load-balancer,
+	// whether the LB's children are directly the nodes internal IPs using the nodePort when the service type is NodePort.
+	// It allows services to be reachable when traefik runs externally from the kubernetes cluster but within the same network of the nodes.
```

Comment:
Please do not forget to regenerate the CRDs.

```suggestion
	// It allows services to be reachable when Traefik runs externally from the Kubernetes cluster but within the same network of the nodes.
```